### PR TITLE
Add tags in results view fix

### PR DESCRIPTION
**Pull request by georgedias**
Fixes issue #5425 
Additional Fixes:
- [x] Added wait cursor and spinner while loading evaluations
- [x] Enabled removing all profiles loaded into the view panel (same as evaluations)
- [x] Update the E2E Update and Delete evaluation test

### CODE REVIEW

**Review by em-c-rod — APPROVED** [review]

File: apps/frontend/src/store/data_filters.ts [hunk 1]
```diff
@@ -264,6 +272,11 @@ export class FilteredData extends VuexModule {
     }
   }
 
+  // check to see if any profile is selected
+  get any_profile_selected(): boolean {
+    return this.selectedProfileIds.length > 0 ? true : false;
```
Comment (Amndeep7) [inline apps/frontend/src/store/data_filters.ts:277]:
```suggestion
    return this.selectedProfileIds.length > 0;
```

**Review by Amndeep7 — APPROVED** [review]

**Comment by Amndeep7** [comment]
Would appreciate that one codesmell fix but not gonna make myself a blocker for it.  Feel free to merge whenever @georgedias

File: apps/frontend/src/components/global/tags/TagRow.vue
Comment (Amndeep7) [inline apps/frontend/src/components/global/tags/TagRow.vue:1]: Actually looks like there's still an issue.

Replication:
1. Save a file to the db
2. Give it a tag from the results view
3. Give it a tag from the db view
4. Give a new tag from the results view - see that the tag you gave in the db view disappeared from the 'already applied' list and when you save this new tag list it'll disappear from the card as well


6. Save the tag list from there - watch your db tag go away unless you refreshed between steps 3 and 4

**Review by Amndeep7 — CHANGES_REQUESTED** [review]

**Comment by Amndeep7** [comment]
https://github.com/mitre/heimdall2/assets/2015821/d5d4504f-9ee0-4541-88ce-df35efcd9c4d

**Comment by Amndeep7** [comment]
I made a heroku deployment to do the testing but it'll probably expire when you look at this again on tuesday

**Review by wdower — APPROVED** [review]

**Review by Amndeep7 — APPROVED** [review]

**Comment by wdower** [comment]
Tested on Heroku; the tagging buttons for both results view and the loading modal are synced appropriately

**Pull request merged**